### ISSUE TITLE
New rule: validate enum

### ISSUE THREAD

**Issue by PeterCervienka** [cicerono/validator#30]
Would be nice, if the validator is able to valid a specific list of items.... example:

We have an enum with values ONCE, MONTH, YEAR and a value for validation is 'ONCE'. This should be a valid value, but 'QUARTEL' isn't valid.

config could be
```
outpaymentFrequency: { enum: { values: ['ONCE'. 'MONTH', 'YEAR'] } }
```

**Comment by relekang** [comment]
Good idea 👍 

What about calling it `options` and support these types:
```javascript
{
  field: { options: [1, 2, 3] }
  otherField: { options: { values: [1, 2, 3], if: () => true } }
}
```

**Comment by PeterCervienka** [comment]
Yes, `options` sounds good. Regarding config, I think the second one is fine. It's flexible enough to add more properties, if it's needed. We do not need to support 2 configs.
